src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -53,7 +53,7 @@ const PopupDrawingToolbar = ({ className, onDelete, onReply, reference }: Props)
                 <button
                     className="ba-PopupDrawingToolbar-delete"
                     data-testid="ba-PopupDrawingToolbar-delete"
-                    onClick={() => onDelete()}
+                    onClick={onDelete}
```

Comment:
I assumed these wrappers were intentional to encapsulate the `event` argument. The callback prop types should be updated to show that they receive it now.

Reply:
It's true I had done it that way, but then typescript wasn't complaining about it. But I'll revert this to be more consistent